ntoskrnl/ps/apphelp.c

```diff
@@ -10,12 +10,12 @@
 /*
 Useful references:
 https://github.com/mandiant/ShimCacheParser/blob/master/ShimCacheParser.py
-http://technet.microsoft.com/en-us/library/dd837644(v=ws.10).aspx
-http://msdn.microsoft.com/en-us/library/bb432182(v=vs.85).aspx
-http://www.alex-ionescu.com/?p=43
-http://recxltd.blogspot.nl/2012/04/windows-appcompat-research-notes-part-1.html
-http://journeyintoir.blogspot.ch/2013/12/revealing-recentfilecachebcf-file.html
-https://dl.mandiant.com/EE/library/Whitepaper_ShimCacheParser.pdf
+https://learn.microsoft.com/en-us/previous-versions/windows/it-pro/windows-7/dd837644(v=ws.10)
+https://learn.microsoft.com/en-us/windows/win32/devnotes/application-compatibility-database
+https://www.alex-ionescu.com/secrets-of-the-application-compatilibity-database-sdb-part-4/
+https://web.archive.org/web/20170101173150/http://recxltd.blogspot.nl/2012/04/windows-appcompat-research-notes-part-1.html
+http://journeyintoir.blogspot.com/2013/12/revealing-recentfilecachebcf-file.html
+https://dl.mandiant.com/EE/library/Whitepaper_ShimCacheParser.pdf (DEAD_LINK)
```

Comment:
```suggestion
https://web.archive.org/web/20150926070918/https://dl.mandiant.com/EE/library/Whitepaper_ShimCacheParser.pdf
```
title: "Leveraging the Application Compatibility Cache in Forensic Investigations"